pkg/cloud/gcp/actuators/machine/reconciler_test.go

```diff
@@ -4,27 +4,37 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/google/uuid"
```

Comment:
Dead code here?

Reply:
The import of the uuid package? It's used in creating the machine object below, right?

Reply:
@bison yes, but we're not consuming that data anywhere anymore.  We don't need to mock that up for the tests.

Reply:
This has all gone - I think I borked because I needed to rebase.